src/index.ts

```diff
@@ -103,6 +106,62 @@ async function monitorCoreRpcConnection(): Promise<void> {
   }
 }
 
+function registerMempoolPromStats(pgEvents: PgStoreEventEmitter) {
```

Comment:
I think this function should be in a separate utility file (helpers, or maybe start one for metrics if you think that's best)

Reply:
Done!